src/parser/Parser.ts

```diff
@@ -1056,7 +1056,14 @@ export class Parser {
                 | Expr.Expression
                 | Stmt.PrintSeparator.Tab
                 | Stmt.PrintSeparator.Space)[] = [];
-            values.push(expression());
+
+            //print statements can be empty, so look for empty print conditions
+            if (isAtEnd() || check(Lexeme.Newline, Lexeme.Colon)) {
+                var emptyStringLiteral = new Expr.Literal(new Int32(1), printKeyword.location);
```

Comment:
I'm a little confused – won't this print `1` every time?  It doesn't seem like an empty string 😃

Reply:
Dangit github updated while I was reviewing